### ISSUE TITLE
WellModel takes only pastas TimeSeries

### ISSUE THREAD

**Issue by raoulcollenteur** [pastas/pastas#184]
Looking at the code of WellModel, I think this model only works with pastas TimeSeries as input , instead of the usual pandas Series.

Lines 727-728
```
#184 get largest std for meanstress
meanstress = np.max([s.series.std() for s in stress])
```

This should be changed, or at least be made clear in the docstrings.

Example
--------

`ps.WellModel([meny.IN['Extraction 3']["values"]], ps.HantushWellModel, "test", [2])
`

returns

```
AttributeError: 'Series' object has no attribute 'series'
```

**Comment by raoulcollenteur** [comment]
The same is true for FactorModel, I will fix that one.

**Comment by dbrakenhoff** [comment]
Fixed by b6ddf79f5e3062b4c5fce9010ce289b9ac0db9ba
